integration-tests/README.md

```diff
@@ -1,33 +1,109 @@
-# HAC AppStudio Integration tests
+# HAC UI Tests
+Functional tests using [Cypress](https://docs.cypress.io/guides/overview/why-cypress)
 
-## Getting started
+## What can be found here
+The important bits are as follows:
 
-### Run tests
+| Path | Description |
+| -- | -- |
+| `cypress.config.ts` | cypress configuration file, contains all the environment and plugin setup |
+| `support` | contains helper files for cypress, like hooks, custom commands, page objects or plugin integration |
+| `tests` | the actual spec files reside here |
+| `utils` | utilities for easier test development, from UI interaction tasks to network requests |
+| `cypress` | test results, including screenshots, video recordings, HTML reports, xunit files, etc. Particularly useful. |
+
+## Running the tests locally
 Prerequisites:
-* Access to AppStudio cluster 
-* (for cleaning namespace before/after test run) Set up kubelogin with appstudio cluster credentials
+* Nodejs (any version that is not pre-historical will do)
+* Access (credentials) to running RHTAP instance  
+* Github token for the `advanced-happy-path` spec (can be obtained from openshift-ci vault for hac-dev, contact `jrichter` for access)
 
-Run `npm run start:prod:beta`, then `../node_modules/.bin/cypress run --env USERNAME=<USERNAME>,PASSWORD=<PASSWORD>` and change USERNAME and PASSWORD or open cypress `npx cypress open` and run tests from GUI (you need to update username and password in integration-tests/plugins/index.ts).
+Certain environment variables will need to be set up, depending on what you intend to run. The most convenient way is to export them from the CLI, in which case they need to be prefixed with `CYPRESS_`, e.g. `USERNAME` is set as `export CYPRESS_USERNAME=username`. Alternatively, they can be passed to cypress via `-e` flag, e.g `npx cypress run -e USERNAME=username`.
 
-You can also use more variables - see the table below:
-| Environment Variable | Description | Default value | Example |
+Find the supported variables in the table below:
+| Variable name | Description | Required | Default Value |
+| -- | -- | -- | -- |
+| `HAC_BASE_URL` | The URL to the main page of RHTAP UI | Yes | 'https://prod.foo.redhat.com:1337/beta/hac/application-pipeline' |
+| `USERNAME` | Username for SSO login | Yes | '' |
+| `PASSWORD` | Password for SSO login | Yes | '' |
+| `PR_CHECK` | Assume the test is a PR check, enable report portal, assume keycloak is used instead of RH-SSO | For PR checks | false |
+| `PERIODIC_RUN` | Assume the test is a nightly run, keycloak is used instead of RH-SSO | For runs using ephemeral RHTAP environment | false |
+| `REMOVE_APP_ON_FAIL` | Clean up applications from the cluster even if tests fail | No | false |
+| `GH_TOKEN` | Github token for network requests | For the `advanced` spec | '' |
 | -- | -- | -- | -- |
```

Comment:
This seems to be Empty Row in between the table.

Reply:
Just wanted a spacer before the report portal variables. Since there is no way to make a fat line, I just used a new row.

Reply:
There is an inconsistency in the table with ty `CYPRESS` prefixes - in the first half they are not used, but they are used fort the Report Portal variables. I would use them also for the first half, as there is a `GH_COMMENTBODY` which is used *without* the CYPRESS prefix and it may cause confusion.